doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -5204,6 +5204,13 @@ Sockets
    ``2`` Tracks IO Buffer Memory and OpenSSL Memory allocations and releases
    ===== ======================================================================
 
+.. ts:cv:: CONFIG proxy.config.system_clock INT 0
+
+   *For advanced users only*. This allows to specify the underlying system clock
+   used by ATS. The default is ``CLOCK_REALTIME`` (``0``), but a higher performance
+   option could be ``CLOCK_REALTIME_COARSE`` (``5``). See ``clock_gettime(2)``_ for
+   more details. On Linux, these definitions can be found in ``<linux/time.h>``.
+
```

Comment:
I think this isn't rendering how we would like it to. Do we want that underscore after the `clock_gettime(2)` reference? Here's the CI built version of the docs:

https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/2600/artifact/output/10163/docbuild/html/admin-guide/files/records.yaml.en.html#proxy-config-system-clock

Reply:
Fixed I hope

Reply:
Looks alright to me:  https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/2604/Sphinx_20Docs/admin-guide/files/records.yaml.en.html#proxy-config-system-clock